client/app.jsx

```diff
@@ -21,9 +22,11 @@ class App extends React.Component {
   componentDidMount() {
 
     var listingId = this.state.listingId
```

Comment:
Very interesting how you don't define a state object initially, just setState within the componentDidMount. Any reason why you chose this pattern? I'm just curious.

Reply:
GitHub has hidden it from us!  those snakes.